openedx/core/openapi.py

```diff
@@ -0,0 +1,20 @@
+"""
+Open API support.
+"""
+
+from rest_framework import permissions
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Open edX API",
```

Comment:
Should this be `edx-platform REST API` since this covers only edx-platform and not all of Open edX?